spec/features/user_views_article_spec.rb

```diff
@@ -14,32 +14,31 @@
   end
 
   let(:subcategory) do
-    FactoryGirl.create(:category, parent_id: category.id)
+    FactoryGirl.create(:category, name: "Subcat", parent: category)
   end
 
   let(:article) do
     FactoryGirl.create(:article, category: subcategory)
   end
 
-  scenario 'without authenticating' do
-    visit article_path(article)
-    subcat = Category.find(article.category.parent_id).name
+  scenario 'user views article' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.